site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -173,12 +186,12 @@ recommended for a production deployment.
 ```bash
 gcloud container node-pools create agones-system \
   --cluster=[CLUSTER_NAME] \
-  --no-enable-autoupgrade \
+  --region=[COMPUTE_REGION] \
   --node-taints agones.dev/agones-system=true:NoExecute \
   --node-labels agones.dev/agones-system=true \
   --num-nodes=1
```

Comment:
I think we should add a machine-type flag here for the system node pool (`e2-standard-4`). We specify one for the metrics node pool but not here which I find strange.